experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -107,6 +107,14 @@ get_field(sim::ClimaAtmosSimulation, ::Val{:gas_constant_air}) =
 
 get_surface_params(sim::ClimaAtmosSimulation) = CAP.surface_fluxes_params(sim.integrator.p.params)
 
+function update_field!(atmos_sim::ClimaAtmosSimulation, ::Val{:co2_gm}, field)
```

Comment:
does "gm" in `co2_gm` here stand for "global mean"?

Reply:
yes, adopting the syntax of ClimaAtmos